tests/test_connections/test_oef/test_models.py

```diff
@@ -68,10 +67,10 @@ def test_query(self):
         query = Query([
             And([
                 Or([
-                    Not(Constraint("foo", Gt(1))),
-                    Not(Constraint("bar", Eq("baz")))
+                    Not(Constraint("foo", ConstraintType("==", 1))),
```

Comment:
This used to be greater than, maybe good to keep that way to add some variety to the test?